api/src/test/java/org/openmrs/test/jupiter/BaseContextSensitiveTest.java

```diff
@@ -131,6 +133,9 @@
 	 */
 	protected static Properties runtimeProperties;
```

Comment:
Should we not also adapt https://github.com/openmrs/openmrs-core/blob/master/api/src/test/java/org/openmrs/test/jupiter/BaseContextSensitiveTest.java#L348-L360

@ibacher ?

I thought we also remove

```java
File tempappdir = File.createTempFile("appdir-for-unit-tests-", "");
			tempappdir.delete(); // so we can make it into a directory
			tempappdir.mkdir(); // turn it into a directory
			tempappdir.deleteOnExit(); // clean up when we're done with tests
```

and the try catch. So we are only left with setting the runtime property and the `OpenmrsUtil.setAppDir`.

Reply:
@teleivo  I completely agree.

Reply:
please remove the try catch 

and only keep the lines

https://github.com/openmrs/openmrs-core/blob/master/api/src/test/java/org/openmrs/test/jupiter/BaseContextSensitiveTest.java#L354-L356

that are inside the try catch.

We are doing this since JUnit creates and deletes the tempappdir. So the only thing we need to do for tests that extend this base class is to set the runtime property and save the app dir using the openmrs utils so tests/code under test can write to this temp dir.